api/db/schema.rb

```diff
@@ -303,6 +303,7 @@
     t.uuid     "creator_id"
     t.uuid     "start_text_section_id"
     t.integer  "position"
+    t.string   "spine",                              array: true
```

Comment:
Right now this field default to nil, rather than to []. Is that the behavior we want?